From 77d8734e1870f842d1e62addec82e5c05c1a6630 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Tue, 29 Oct 2019 16:07:57 +0200
Subject: [PATCH] sdk_dpaa: ethtool: drop unused variable

Remove unused variable that was missed in past code refactorings.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c
@@ -90,7 +90,6 @@ static char dpa_stats_global[][ETH_GSTRI
 static int __cold dpa_get_ksettings(struct net_device *net_dev,
 		struct ethtool_link_ksettings *cmd)
 {
-	int			 _errno;
 	struct dpa_priv_s	*priv;
 
 	priv = netdev_priv(net_dev);
@@ -106,7 +105,7 @@ static int __cold dpa_get_ksettings(stru
 
 	phy_ethtool_ksettings_get(priv->mac_dev->phy_dev, cmd);
 
-	return _errno;
+	return 0;
 }
 
 static int __cold dpa_set_ksettings(struct net_device *net_dev,
